src/Microsoft.Health.Fhir.SqlServer/Features/Storage/ProcessingFlag.cs

```diff
@@ -0,0 +1,70 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Storage
+{
+    internal class ProcessingFlag<TLogger>
```

Comment:
Why is `ProcessingFlag` class using another class's logger?

Reply:
I just kept current behavior when moved ProcessingFlag from private to internal. Does it hurt?